next.config.mjs

```diff
@@ -1,7 +1,17 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
     output: "standalone",
-    basePath: process.env.NEXT_PUBLIC_BASE_PATH
+    basePath: process.env.NEXT_PUBLIC_BASE_PATH,
+    images: {
+        remotePatterns: [
+            {
+                protocol: 'https',
+                hostname: 'www.nb.no',
+                port: '',
+                pathname: '/services/image/resolver/**'
```

Comment:
Er det dette endepunktet vi egentlig kommer til å bruke for å hente bildene, eller dette midlertidig? Isåfall er det fint med en kommentar, som i `route.ts`

Reply:
Det er nok også midlertidig til vi har avklart hvor vi skal hente fra. Legger på kommentar.